pkg/storage/runtime.go

```diff
@@ -167,7 +167,11 @@ func (r *runtimeService) createContainerOrPodSandbox(systemContext *types.System
 			ref, err = istorage.Transport.ParseStoreReference(r.image.GetStore(), otherRef.DockerReference().FullName())
 		}
 		if err != nil {
-			return ContainerInfo{}, err
+			// maybe it's just imageID
+			ref, err = istorage.Transport.ParseStoreReference(r.image.GetStore(), "@"+imageID)
+			if err != nil {
+				return ContainerInfo{}, err
+			}
 		}
```

Comment:
It looks like this makes the logic what was previously at lines 174-180 unnecessary.